config/test.yaml

```diff
@@ -0,0 +1,5 @@
+postgres:
+  user: postgres
+  password: postgres
+  dbname: iterable_test
```

Comment:
I think you can add a `TestMain` to the db_test file. Look here for more info https://golang.org/pkg/testing/.